source/fuzz/fuzzer_util.cpp

```diff
@@ -1342,7 +1342,47 @@ opt::Instruction* GetLastInsertBeforeInstruction(opt::IRContext* ir_context,
   return CanInsertOpcodeBeforeInstruction(opcode, &*it) ? &*it : nullptr;
 }
 
-}  // namespace fuzzerutil
+uint32_t MaybeGetLocalVariable(opt::IRContext* ir_context, uint32_t type_id,
+                               uint32_t function_id) {
+  uint32_t return_id = 0;
+  auto function = FindFunction(ir_context, function_id);
```

Comment:
Make sure to assert that `type_id` and `function_id` have valid values. That is:
1. `function != nullptr`
2. see my comment in the header file regarding the conditions on the `type_id`.